OneSignalSDK/onesignal/src/main/java/com/onesignal/OneSignal.java

```diff
@@ -1012,19 +1012,6 @@ public static boolean requiresUserPrivacyConsent() {
     * This method will be replaced by remote params set
     */
    public static void setRequiresUserPrivacyConsent(final boolean required) {
-      if (taskController.shouldQueueTaskForInit(OSTaskController.SET_REQUIRES_USER_PRIVACY_CONSENT)) {
```

Comment:
what will happen if the user put setRequiresUserPrivacyConsent as false, and then remote params comes with requiresUserPrivacyConsent as true?

Reply:
Good catch, this does work correctly by respecting the remote param
https://github.com/OneSignal/OneSignal-Android-SDK/pull/1153/commits/eaf2db6b1bbac5ad34ac59e146989a7e12f0308d